src/workloads/query/TelemetryQueryShapes.yml

```diff
@@ -0,0 +1,999 @@
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {a: {$not: {$exists: true}}}
+
+- Name: FindLtGt
+  Type: CrudActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {b: {$lt: 6}, a: {$gt: 4}}
+
+- Name: AggMatchGroup
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$match: {a: 42}}, {$group: {_id: "$b", sum: {$sum: "$a"}}}]
+
+- Name: AggCondGte
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$project: {number: {$cond: [{$gte: ["$a", 250]}, 30, 20]}}}]
+
+- Name: AggToLowerStr
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$project: {item: { $toLower: "$string"}}}]
+
+- Name: AggToLowerArr
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$project: {item: {$toLower: ["$string"]}}}]
+
+- Name: AggGetField
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$match: {$expr: {$lt: [{$getField: {$literal: "$a"}}, 30]}}}]
+
+# Same Query Shape.
```

Comment:
I think that you can just specify `Threads: 16` instead of repeating it - right?